debian/control

```diff
@@ -21,7 +21,7 @@ Build-Depends: cmake (>= 3.5),
                gperf,
                g++ (>= 7),
                javahelper,
-# Make-Check   jq,
```

Comment:
these are unrelated changes, can you remove them

Reply:
@ideepika I dont know how I got them may be during rebasing something happened I will remove them

Reply:
@suab321321 this might be because you had run make check and then committed all changes, please read about `git add -p` to selectively add changes.